### ISSUE TITLE
[Virtual Hydlide] Corruption de texture en noyau compute shader

### ISSUE THREAD

**Issue by fafling** [FCare/Kronos#601]
Dans la WIP du 24/08, avec le noyau compute shader uniquement, la texture de certains sprites est corrompue de manière intermittente.
![Capture_virtual_hydlide_corruptions_rasterizer_20190824](https://user-images.githubusercontent.com/7624262/63901135-a7c61500-ca03-11e9-8651-8f651c25e668.PNG)

Un save state à cet endroit :
[virtual_hydlide_corruption_raterizer.zip](https://github.com/FCare/Kronos/files/3553347/virtual_hydlide_corruption_raterizer.zip)

Testé avec le CD US.

**Comment by FCare** [comment]
Chez moi c'est nickel... 32 bits, nvidia, windows?

**Comment by fafling** [comment]
Oui, tout ça. Le problème est toujours présent dans la WIP du 02/09.

**Comment by fafling** [comment]
Toujours présent en WIP du 01/12.

**Comment by fafling** [comment]
Toujours présent en WIP du 20/12 (64 bits maintenant).
